packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -193,11 +193,13 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     this.setState({ suggestionsVisible: false });
   };
 
-  public completeSuggestion() {
```

Comment:
If the signature for this is changed why are we adding the method in the `.types.ts` file instead of changing the signature?